CMakeLists.txt

```diff
@@ -185,12 +185,63 @@ endif()
 #add_subdirectory(iverilog)
 add_subdirectory(libs)
 add_subdirectory(libopenfpga)
-add_subdirectory(yosys)
+#add_subdirectory(yosys)
 add_subdirectory(abc)
 add_subdirectory(ace2)
 add_subdirectory(vpr)
 add_subdirectory(openfpga)
 
+# Compilation options for yosys
+include(CMakeParseArguments)
+
+##project(yosys)
+
+# Version number 
+set(YOSYS_VERSION_MAJOR 0.7)
```

Comment:
We were using v0.9 release and I do not know if the version number here matters or not?

Reply:
I think this is not required now, I removed these variables.